### PR TITLE
Model card for bert-turkish-question-answering question-answering model

### PR DESCRIPTION
**Pull request by lserinol**
First model card for bert-turkish-question-answering question-answering model

### CODE REVIEW

File: model_cards/lserinol/bert-turkish-question-answering/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+language: Turkish
```
Comment (julien-c) [inline model_cards/lserinol/bert-turkish-question-answering/README.md:2]:
```suggestion
language: turkish
```

**Comment by julien-c** [comment]
Awesome, thanks for sharing! Did you fine-tune https://huggingface.co/dbmdz/bert-base-turkish-cased?

 cc @stefan-it

**Comment by julien-c** [comment]
[Model page](https://huggingface.co/lserinol/bert-turkish-question-answering)

**Comment by lserinol** [comment]
> Awesome, thanks for sharing! Did you fine-tune https://huggingface.co/dbmdz/bert-base-turkish-cased?
> 
> cc @stefan-it

Yes

**Pull request merged**